Sources/Private/NodeRenderSystem/Nodes/PathNodes/PolygonNode.swift

```diff
@@ -88,14 +88,33 @@ final class PolygonNode: AnimatorNode, PathNode {
   }
 
   func rebuildOutputs(frame: CGFloat) {
-    let outerRadius = properties.outerRadius.value.cgFloatValue
-    let outerRoundedness = properties.outerRoundedness.value.cgFloatValue * 0.01
-    let numberOfPoints = properties.points.value.cgFloatValue
-    let rotation = properties.rotation.value.cgFloatValue
-    let position = properties.position.value.pointValue
+    let path = BezierPath.polygon(
+      position: properties.position.value.pointValue,
+      numberOfPoints: properties.points.value.cgFloatValue,
+      outerRadius: properties.outerRadius.value.cgFloatValue,
+      outerRoundedness: properties.outerRoundedness.value.cgFloatValue,
+      rotation: properties.rotation.value.cgFloatValue,
+      direction: properties.direction)
 
+    pathOutput.setPath(path, updateFrame: frame)
+  }
+
+}
+
+extension BezierPath {
+  /// Creates a `BezierPath` in the shape of a polygon
+  static func polygon(
+    position: CGPoint,
+    numberOfPoints: CGFloat,
+    outerRadius: CGFloat,
+    outerRoundedness inputOuterRoundedness: CGFloat,
+    rotation: CGFloat,
+    direction: PathDirection)
+    -> BezierPath
+  {
     var currentAngle = (rotation - 90).toRadians()
     let anglePerPoint = ((2 * CGFloat.pi) / numberOfPoints)
+    let outerRoundedness = inputOuterRoundedness * 0.01
```

Comment:
nit: document this magic number?

Reply:
Not reaally sure why this is necessary -- it was here before (https://github.com/airbnb/lottie-ios/blob/master/Sources/Private/NodeRenderSystem/Nodes/PathNodes/PolygonNode.swift#L92)

Reply:
Ah ok—wonder if it's converting from percentage to fractional again?